### ISSUE TITLE
R2 PUT is returning "null" even though its technically working

### ISSUE THREAD

**Issue by saturnonearth** [chientrm/wrangler-proxy#18]
When I am running the proxy, the "PUT" command for R2 is not returning anything. It's returning `null` but on CF it returns an object.

```js
const file = form.get('file');
const image = await locals.R2.put(`${fileName}.${ext}`, file);

if (!image) {
error(500, {
  title: 'Failed to upload image.',
  detail: 'Something went wrong when trying to save your save your image.'
});
}
```

This code fails because `image` is null, even though it gets properly uploaded via proxy bindings - but does not fail when hosted on CF.

**Comment by chientrm** [comment]
Fixed by release [2.2.4](https://github.com/chientrm/wrangler-proxy/releases/tag/2.2.4)

**Comment by saturnonearth** [comment]
> Fixed by release [2.2.4](https://github.com/chientrm/wrangler-proxy/releases/tag/2.2.4)

Thank you lol I thought I was going crazy here
